extensions/amp-story/1.0/amp-story.css

```diff
@@ -176,6 +176,26 @@ amp-story-page[active] {
   z-index: 1 !important;
 }
 
+amp-story amp-sidebar[side][open] {
+  background-color: #ffffff !important;
+  max-width: 280px !important;
+  min-width: 280px !important;
+  width: 280px !important;
+  display: block !important;
+}
+
+amp-story amp-sidebar[side=right] {
```

Comment:
Do we need to duplicate this block?

Reply:
Ah, thanks for the catch. This was originally added for setting transitions for the the sidebar closing. Now that we are decoupling those changes with the mask/background changes, I will remove this.